rpcs3/Emu/RSX/GL/GLRenderTargets.cpp

```diff
@@ -234,17 +218,12 @@ void GLGSRender::init_buffers(rsx::framebuffer_creation_context context, bool sk
 
 	const u8 color_bpp = get_format_block_size_in_bytes(layout.color_format);
 	const u8 depth_bpp = (layout.depth_format == rsx::surface_depth_format::z16 ? 2 : 4);
```

Comment:
I think the variable `depth_bpp` can be removed unless you want to replace:
```
m_depth_surface_info.bpp = (layout.depth_format == rsx::surface_depth_format::z16? 2 : 4);
```
by
```
m_depth_surface_info.bpp = depth_bpp;
```